frontend/src/types/ServerConfig.ts

```diff
@@ -111,6 +111,7 @@ export interface ToleranceConfig {
 
 export interface ServerConfig {
   accessibleNamespaces: Array<string>;
+  ambientProfile: boolean;
```

Comment:
another one of these "profile" names that I think needs to change -- "enabled" or "supported" are more preferable to me.

Reply:
This was also renamed to "enabled"